shared/src/db/database.ts

```diff
@@ -12,6 +12,7 @@ import {
     TagDto,
     TagType,
     Uuid,
+    luminaryInternalsDto,
```

Comment:
A type is normally spelled starting with a capital letter. As it is not a type used to communicate data to external systems, it is not really a DTO.
```suggestion
    LuminaryInternals,
```